### PR TITLE
Implemented: Updated UI of Settings page(#32j3r6t)

### LINKED ISSUE #124: Update Settings page with updated UI
Reference - https://www.figma.com/file/bVPRRw282CqGKMdbz7dciH/HC-Ionic-design-system?node-id=23579%3A83419&t=EwnGrKo71tXMohaa-0

### PR DESCRIPTION
**Pull request by shashwatbangar**
### Related Issues

Closes #124 

### Screenshots of Visual Changes before/after (If There Are Any)

Screenshot 2023-01-09 at 6 12 11 PM

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)
- [ ] I have changed the target to hacktoberfest branch

### CODE REVIEW

File: src/views/Settings.vue [hunk 1]
```diff
@@ -126,4 +169,20 @@ export default defineComponent({
...
+  }
+  section {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+    align-items: start;
+  }
+  .user-profile {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(400px, 1fr));
+  }
```
Comment (k2maan) [inline src/views/Settings.vue:187]: Fix indentation

**Review by k2maan — APPROVED** [review]

File: src/views/Settings.vue (hunk 1 above)
Comment (shashwatbangar) [reply src/views/Settings.vue]: Done sir

File: src/views/Settings.vue [hunk 2]
```diff
@@ -7,45 +7,84 @@
...
-        <!-- Profile of user logged in -->
-        <ion-item>
-          <ion-icon :icon="personCircleOutline" slot="start" />
-          <ion-label>{{ userProfile !== null ? userProfile.partyName : '' }}</ion-label>
-         <ion-button slot="end" fill="outline" color="dark" @click="logout()">{{ $t("Logout") }}</ion-button>
-        </ion-item>
-      </ion-list>
+        <ion-card>
+          <ion-card-header>
+            <ion-card-subtitle>
```
Comment (adityasharma7) [inline src/views/Settings.vue:48]: I think this is not needed in this app

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Settings.vue (hunk 2 above)
Comment (shashwatbangar) [reply src/views/Settings.vue]: ok sir

**Pull request merged**